docs/build.rst

```diff
@@ -0,0 +1,122 @@
+..
+   Author: Adam Turner
+
+
+Building PEPs Locally
+=====================
+
+Whilst editing a PEP, it is useful to review the rendered output locally. This
+can also be used to check that the PEP is valid reStructuredText before
+submission to the PEP editors.
+
+The rest of this document assumes you are working from a local clone of the
+`PEPs repository <https://github.com/python/peps>`__, with Python 3.9 or later
+installed.
+
+
+Render PEPs locally
+-------------------
+
+1. Install requirements
+
+   .. code-block:: console
+
+      $ python -m venv venv
+      $ . venv/bin/activate
```

Comment:
Maybe I overthought this -- I thought `source` was only a synonym for `.` on `bash` &c -- not actually in the POSIX specification. 

Equally, I use Windows so I'm not really impacted -- if using `source` would be better here then let's do so. 

On second thoughts, perhaps we should just redirect to the `venv` docs -- covering every corner case will add a high degree of verbosity when *most* readers will be familiar with virtual environments, and the official documentation will cover the things a two line invocation will miss.

A

Reply:
I don’t think `source` would be better.

Reply:
> On second thoughts, perhaps we should just redirect to the `venv` docs -- covering every corner case will add a high degree of verbosity when _most_ readers will be familiar with virtual environments, and the official documentation will cover the things a two line invocation will miss.

Yeah, I agree this is probably a better idea. While including the commands explicitly here can be faster and easier for users, in this case I think we can expect our users to have enough initiative, general Python knowlege and technical proficiency to read the venv docs and be able to apply them here at their discretion (for example, I use conda envs for this instead of venvs, and other users might use `virtualenvwrapper` or other tools).

Reply:
Yep, I prefer conda/mamba personally (dynamically resolving the Python version!), but I think the PEP docs should use the standard library where possible. Updated to refer to the PUG.

A